vmdb/app/models/miq_report/search.rb

```diff
@@ -14,7 +14,7 @@ def get_limit_offset(page, per_page)
   end
 
   def get_sqltable(assoc)
-    r = self.db_class.reflect_on_association(assoc.to_sym)
+    r = db_class.reflection_with_virtual(assoc.to_sym)
```

Comment:
> The caller is get_col_info which has the guard if apply_sortby_in_search where apply_sortby_in_search = !self.db_class.table_name.nil?. Do ActsAsArModel records have a table_name? If not, then how did an ActsAsArModel make its way to get_sqltable?

Good catch.  They definitely didn't have table names.  I should revert the change to `db_class.reflection_with_virtual` and fix this conditional. I wonder if there are other classes that aren't `ActiveRecord::Base` or `ActsAsArModel` but don't have a table name.

I can't really nil the table name in `VmdbDatabaseConnection` since Rails uses that. :(